### PR TITLE
Add support for additional fields from V2 ALB logs

### PR DESCRIPTION
**Pull request by kaiyan-sheng**
This PR is to add support for additional fields from V2 ALB logs into integrations. 
PR in Beats: https://github.com/elastic/beats/pull/21540

### CODE REVIEW

File: packages/aws/data_stream/elb_logs/elasticsearch/ingest_pipeline/default.yml [hunk 1]
```diff
@@ -29,7 +29,7 @@ processors:
           %{TIMESTAMP_ISO8601:event.start}
           \"(?:-|%{DATA:_tmp.actions_executed})\"
           \"(?:-|%{DATA:aws.elb.redirect_url})\"
-          \"(?:-|%{DATA:aws.elb.error.reason})\"
+          \"(?:-|%{DATA:aws.elb.error.reason})\"( \"(?:-|%{DATA:_tmp.target_port})\")?( \"(?:-|%{DATA:_tmp.target_status_code})\")?( \"(?:-|%{DATA:aws.elb.classification})\")?( \"(?:-|%{DATA:aws.elb.classification_reason})\")?
```
Comment (mtojek) [inline packages/aws/data_stream/elb_logs/elasticsearch/ingest_pipeline/default.yml:32]: FYI there is support for pipeline tests to verify such changes. You can find sample ones in nginx integration (just a side note, not blocking).

**Review by mtojek — APPROVED** [review]
LGTM!

File: packages/aws/data_stream/elb_logs/elasticsearch/ingest_pipeline/default.yml (hunk 1 above)
Comment (kaiyan-sheng) [reply packages/aws/data_stream/elb_logs/elasticsearch/ingest_pipeline/default.yml:32]: Thank you for the suggestion! I'm adding pipeline tests here just to make sure it works as expected.

**Comment by kaiyan-sheng** [comment]
@mtojek [CI failed](https://beats-ci.elastic.co/blue/organizations/jenkins/Beats%2Fintegrations/detail/PR-304/5/pipeline#step-109-log-622) with the same error I got when I tried to run `elastic-package test pipeline` locally:
```
+        },
+        {
+            "message": "",
+            "error": {
+                "message": "Provided Grok expressions do not match field value: []"
+            }
         }
     ]
 }
```

**Comment by mtojek** [comment]
I've finished the investigation. It' a bug in elastic-package. As a temporary workaround you can use the following config:

```json
{
  "multiline": {
    "first_line_pattern": "^http"
  }
}
```

but I will work on the workaround today and post the update here.

**Comment by mtojek** [comment]
@ycombinator that's a weird HTTP error during system tests: 

```
[2020-10-12T15:49:03.131Z] Error: error running package system tests: could not add data stream config to policy: could not add package to policy; API status code = 502; response body = {"statusCode":502,"error":"Bad Gateway","message":"'404 Not Found' error response from package registry at https://epr.elastic.co/package/apache/0.2.4/"}
```

**Comment by mtojek** [comment]
jenkins run the tests please

**Comment by kaiyan-sheng** [comment]
Merging, CI is happy now.

**Pull request merged**